experimenter/experimenter/experiments/models.py

```diff
@@ -665,11 +665,16 @@ def can_review(self, reviewer):
     # Results are available if enrollment is complete and
     # more than a week has passed after that.
     @property
-    def results_ready(self):
+    def results_ready_date(self):
         if self.proposed_enrollment_end_date:
-            resultsReadyDate = self.proposed_enrollment_end_date + datetime.timedelta(
+            return self.proposed_enrollment_end_date + datetime.timedelta(
                 days=NimbusConstants.DAYS_UNTIL_ANALYSIS
             )
+
+    @property
+    def results_ready(self):
+        if self.proposed_enrollment_end_date:
+            resultsReadyDate = self.results_ready_date
```

Comment:
Camel case in Python?  🤔 🤔 🤔

Reply:
Whoops, going back and forth in TS and Python...

:camel: :snake: